projects/storefrontlib/src/cms-components/myaccount/my-interests/my-interests.component.spec.ts

```diff
@@ -86,180 +88,137 @@ const mockedInterests: ProductInterestList = {
         },
       ],
     },
+    {
+      product: {
+        code: '553638',
+        name: 'NV11',
+        images: {
+          PRIMARY: {
+            thumbnail: {
+              altText: 'NV11',
+              format: 'thumbnail',
+              imageType: ImageType.PRIMARY,
+              url: 'image-url',
+            },
+          },
+        },
+        price: {
+          formattedValue: '$188.69',
+        },
+        stock: {
+          stockLevel: 0,
+          stockLevelStatus: 'outOfStock',
+        },
+      },
+      productInterestEntry: [
+        {
+          dateAdded: new Date(),
+          interestType: 'BACK_IN_STOCK',
+        },
+      ],
+    },
   ],
```

Comment:
product with variant should be added

Reply:
the API interface does not return the variant related attributes.